docs/UserGuide.md

```diff
@@ -108,18 +113,28 @@ If you would like to search the database, you can do so in a few ways:
 3. Find Q&A with **tags** containing any of the given keywords. 
     Keywords for tags can also have phrases, however this requires the text to have an exact match (case-insensitive).
 
-    Format: `find a/KEYWORD`, for 2 or more words: `find q/KEYWORD1, KEYWORD2`
+    Format: `find a/KEYWORD`, for 2 or more words: `find q/PHRASE1, KEYWORD1`
 
 4. Find Q&A with **questions or answers** containing any of the given keywords.
 
-    Format: `find a/KEYWORD`, for 2 or more words: `find q/KEYWORD1, KEYWORD2`
+    Format: `find a/KEYWORD`, for 2 or more words: `find q/PHRASE, KEYWORD1`
 
 Examples:
-* `find q/system, disease` - finds questions with the word "system" and/or "disease".
-* `find a/vessels, chronic disease` - finds answers with the word vessels, and/or the phrase "chronic disease".
+* `find q/system, disease` - finds questions containing the word "system" and/or "disease".
+* `find a/vessels, chronic disease` - finds answers containing the word vessels, and/or the phrase "chronic disease".
 * `find t/anatomy, Nervous System` - finds question sets tagged with the word "anatomy" and/or "Nervous System" (can be both), requires exact match of words (case-insensitive).
-* `find infection, nervous system` - finds question and/or answers with the word infection, and/or the phrase "nervous system"
+* `find infection, nervous system` - finds question and/or answers containing the word "infection", and/or the phrase "nervous system"
+
+How it should look like in the application:
+
+![FindContext](images/FindContext.PNG)
+
+Expected Result:
+
+![FindResult](images/FindResult.PNG)
 
+The input `find t/Chronic diseases, Diagnosis` will filter out the questions with the tag "Chronic Diseases" and/or "Diagnosis"
```

Comment:
"filter out" sounds like they're being omitted from the search? Maybe can change to "will return questions"

Reply:
aight changed